### ISSUE TITLE
Increase visibility on elements that fail WCAG contrast guidelines

### ISSUE THREAD

**Issue by cupoftwee** [cupoftwee/sarahrobin#50]
<img width="1302" alt="image" src="https://github.com/cupoftwee/sarahrobin/assets/38168454/7ca99a9c-e7d1-4815-b266-20656f39e4e3">
